engine/src/flutter/flow/compositor_context.h

```diff
@@ -138,6 +139,10 @@ class CompositorContext {
                                 bool ignore_raster_cache,
                                 FrameDamage* frame_damage);
 
+    int64_t GetFlutterViewId() const {
```

Comment:
`GetFlutterViewId` is not used anywhere. Unless this is temporary (i.e. it will be used elsewhere in the future), then `CompositorContext` will not need the extra `flutter_view_id` parameter.